### PR TITLE
autofeature for CDI 3 and microprofile context propagation

### PR DESCRIPTION
**Pull request by njr-11**
Create autofeature that will be needed for CDI context for MicroProfile Context Propagation for the new Jakarta CDI 3.0

### CODE REVIEW

**Comment by njr-11** [comment]
#build
#run-libby-bot

**Comment by njr-11** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.cdi3.0-mpContextPropagation1.0.feature [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+symbolicName=com.ibm.websphere.appserver.cdi3.0-mpContextPropagation1.0
+visibility=private
+IBM-Provision-Capability: \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.cdi-3.0))", \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.mpContextPropagation-1.0)(osgi.identity=com.ibm.websphere.appserver.mpContextPropagation-1.1)))"
+-bundles=\
+  com.ibm.ws.cdi.mp.context.jakarta
+IBM-Install-Policy: when-satisfied
+kind=noship
+edition=core
```
Comment (aguibert) [inline dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.cdi3.0-mpContextPropagation1.0.feature:11]: should be `edition=full` when `kind=noship`

**Comment by njr-11** [comment]
#build
#run-libby-bot

**Pull request merged**